README.md

```diff
@@ -13,6 +13,15 @@ Using Go
 go get -u github.com/Gituser143/cryptgo
 ```
 
+Using Docker
+--------
+
+```bash
+docker build -t cryptgo .
+
+docker run -it cryptgo
```

Comment:
```suggestion
docker run -it -v "$HOME:/root" cryptgo
```

It would be better to mount the home directory as a volume as the favourites are stored as a `JSON` file on Disk. Sorry I forgot to mention this earlier

Reply:
No problem